.githooks/pre-commit

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
How does this work on windows do you know, else I can RTFM 🤔 👀

Reply:
No idea. I haven't checked on my PC